xcodeproj/internal/opts.bzl

```diff
@@ -526,16 +569,19 @@ def _process_compiler_opts(
         *   `quotes_includes`: A `list` of quote include paths parsed.
         *   `includes`: A `list` of include paths parsed.
     """
-    conlyopts, cxxopts, search_paths = _process_copts(
+    conlyopts, cxxopts, c_search_paths = _process_copts(
```

Comment:
Since we have to merge with the swift one later, can we change `_process_copts()` to return `conly_search_paths, cxx_search_paths` and merge them all at once?